en/docs/references/api-path.md

```diff
@@ -0,0 +1,57 @@
+# API Path
+
+When you create a service, you need to configure an API to invoke it. This configuration includes the **Path** field where you enter the path to be used by the API to trigger the service as well as to pass path parameters to it.
+
+![Path field](../assets/img/references/path/path-field.png){.cInlineImage-half}
+
+e.g., Create a service named sample with an API trigger.
```

Comment:
```suggestion
e.g., Create a service named `sample` with an API trigger.
```

Reply:
Isn't `sample` is a  too generic name to refer a sample. It will get confused with the noun sample.

Reply:
@RukshiW Seems this is not fixed yet.

Reply:
Fixed it.